### PR TITLE
azurerm_mssql_database - support for the recovery_point_id & restore_long_term_retention_backup_id  property

### PR DESCRIPTION
**Pull request by katbyte**

### CODE REVIEW

File: internal/services/mssql/mssql_database_resource.go [hunk 1]
```diff
@@ -1004,6 +1012,10 @@ func resourceMsSqlDatabaseUpdate(d *pluginsdk.ResourceData, meta interface{}) er
 		props.RecoverableDatabaseId = pointer.To(d.Get("recover_database_id").(string))
 	}
 
+	if d.HasChange("recovery_point_id") {
+		props.RecoveryServicesRecoveryPointId = pointer.To(d.Get("recover_database_id").(string))
```
Comment (catriona-m) [inline internal/services/mssql/mssql_database_resource.go:1016]: should this be `recovery_point_id`?

```suggestion
		props.RecoveryServicesRecoveryPointId = pointer.To(d.Get("recovery_point_id").(string))
```

**Review by catriona-m — APPROVED** [review]
Just left one query inline, otherwise LGTM!

**Comment by katbyte** [comment]
note: this pr should get a acceptance test added to it that demonstrates and tests how to use the new property

**Review by mbfrahry — APPROVED** [review]
LGTM!

**Pull request merged**